src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        String[] colors = {"red", "green", "blue", "yellow", "orange", "purple"};
+        Random random = new Random();
```

Comment:
```suggestion
public class ColorSupplier {
    private static final String[] colors = {"red", "green", "blue", "yellow", "orange", "purple"};
    private final Random random = new Random();
    
    public String getRandomColor() {
```
why creating a new array and new Random each time this method is called